src/DummyData/McSpicy.ts

```diff
@@ -0,0 +1,81 @@
+import { Option, FoodItem } from "./DataTypes";
+
```

Comment:
You cannot possible create a new file for each food item. You need to create a normalised data store for all the food objects and then filter the data from there. Otherwise there would just be too many food files

Reply:
With the actual DB, I would receive a single foodItems object containing all the data. So this is a hardcoded database. Will change when we have actual DB.